src/urllib3/connection.py

```diff
@@ -30,11 +30,12 @@ class BaseSSLError(BaseException):
     SubjectAltNameWarning,
     SystemTimeWarning,
 )
-from .packages.ssl_match_hostname import CertificateError, match_hostname
 from .util import SKIP_HEADER, SKIPPABLE_HEADERS, connection
 from .util.ssl_ import (
+    CertificateError,
     assert_fingerprint,
     create_urllib3_context,
+    match_hostname,
```

Comment:
You can directly use `from ssl import CertificateError, match_hostname` under `import ssl` above

Reply:
In other words, we no longer need the code in our repo since this code is now in the `ssl` module